.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2.1
+
+orbs:
+  cdk: helix/cdk@dev:e52ed15 #0.5.0
```

Comment:
`@dev:alpha` will use the last built alpha version. It's kinda like `latest`. :)

Reply:
is there a `latest` tag? 🤩  would love to have that everywhere.